torchsynth/util.py

```diff
@@ -50,6 +52,29 @@ def normalize(signal: Signal) -> Signal:
     return signal / max_sample
 
 
+def format_hyperparameter_json(hyperparameter_json: str) -> str:
+    """
+    Return formatted JSON string of hyperparameter dictionary -- this removes
+    the new lines and indentation on the hyperparameter name for nicer printing.
+
+    E.g.
+        "name": [
+          "adsr_1",
+          "attack",
+          "curve"
+        ],
+
+    Becomes:
+        "name": ["adsr_1", "attack", "curve"],
+    """
+    hp = re.sub(
+        r'("name": \[)(\n\s+)(.*)(\n\s+)(.*)(\n\s+)(.*)(\n\s+)',
+        r"\1\3 \5 \7",
+        hyperparameter_json,
```

Comment:
Oh god I think this a bit scary. If you're going to do this by hand, then in this function add a proper assert:

```
assert hyperparameter_json == json.loads(hp), "Your hyperparameters cannot easily be converted to JSON using our hacky code"
```

or some such. Like what if some nerd decide to add quotes or commas to their param names?

Reply:
or maybe a try except and then if except log the message and say we're using ugly verbose standard JSON

Reply:
Yah you know what I'm not sure how necessary it is to have this in the library -- if we want we could create a little util script or something to convert json files to a more clean format. Or have an arg that applies this if we think it would be helpful. But I'd be just as happy to ditch this.

Reply:
Let's ditch it and do it by hand when we commit to the repo?